### PR TITLE
Add CRDB-CLI Commands

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/rs/references/crdb-cli-reference.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+The `crdb create` command supports several additional options:
+
+|Flag and argument| Argument type| Description|
+|---|---|---|
+|`no-wait`| no arguments| Prevents CRDB-CLI from running another command before this command finishes|
+|`compression (0-6)`| integer| The level of data compression: 0=Compression disabled| 1=Low compression and resource load| 6=High compression and resource load (Default: 3)|
+|`causal-consistency true`| boolean| Applies updates to all instances in the order they were received|
+|`password <password>`| string| Password for access to the database|
+|`replication true`| boolean| Enables [database replication]({{< relref "/rs/concepts/high-availability/replication.md" >}})| where every master shard replicates to a replica shard (We recommend that you use replication so that active-active database synchronization traffic is off-loaded to the slave shard)|
```
Comment (AlonMagrafta) [inline content/rs/references/crdb-cli-reference.md:104]: I would say "... (We **highly** recommend that you use replication..."

File: content/rs/references/crdb-cli-reference.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+- 1 shard
+- Shard key regex: `{u'regex': u'.*\\{(?<tag>.*)\\}.*'}, {u'regex': u'(?<tag>.*)'}`
+
+The `crdb create` command supports several additional options:
+
+|Flag and argument| Argument type| Description|
+|---|---|---|
+|`no-wait`| no arguments| Prevents CRDB-CLI from running another command before this command finishes|
+|`compression (0-6)`| integer| The level of data compression: 0=Compression disabled| 1=Low compression and resource load| 6=High compression and resource load (Default: 3)|
+|`causal-consistency true`| boolean| Applies updates to all instances in the order they were received|
```
Comment (AlonMagrafta) [inline content/rs/references/crdb-cli-reference.md:102]: We should link here:
https://docs.redislabs.com/latest/rs/administering/database-operations/causal-consistency-crdb/

File: content/rs/references/crdb-cli-reference.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+[-- shards-count <number_of_shards>] /
+[--shard-key-regex <regex_rule>] /
+[--oss-sharding BOOLEAN] /
+[--oss-cluster true]
+[--bigstore true]
+[--bigstore-ram-size <maximum_memory>]
+[--with-module <module_name>]
+```
+
+After you run the command, the response shows the GUID of the task and the GUID of the Active-Active database that was created:
```
Comment (AlonMagrafta) [inline content/rs/references/crdb-cli-reference.md:77]: should we say what GUID stand for?
Globally Unique Identifier

Comment (AlonMagrafta) [reply content/rs/references/crdb-cli-reference.md:77]: Maybe in line 135

File: content/rs/references/crdb-cli-reference.md [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+|`memory-size <ram_limit>`|string| Maximum memory in bytes| kilobytes (kb)| or gigabytes (gb)|
+|`causal-consistency <true | false>`|boolean| Database updates are applied to all instances in the order they were received|
+|`encryption <true | false>`| binary| Enable encryption|
+|`compression (0-6)`| integer| The level of compression of data: 0=Compression disabled| 1=Low compression and resource load| 6=High compression and resource load (Default: 3)|
+|`credentials id=<instance_id>,username=<username>,password=<password>`|string|Update the credentials for the participating cluster|
+|`featureset-version <true | false>`|boolean|Update to latest FeatureSet version|
+|`oss-cluster <true | false>`|boolean|Enable or disable OSS cluster mode|
+|`bigstore <true | false>`|boolean|Use Redis on Flash to add flash memory to the database|
+|`bigstore-ram-size <ram_limit>`|string|RAM limit for RoF database (bytes, MB, or GB)|
+|`with-module <module_name>`|text|The name of the module to add to the database|
```
Comment (AlonMagrafta) [inline content/rs/references/crdb-cli-reference.md:185]: What about referencing to a page where we say which modules are supported?

File: content/rs/references/crdb-cli-reference.md [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+[--replication true] /
+[--encryption true] /
+[--sharding false] /
+[-- shards-count <number_of_shards>] /
+[--shard-key-regex <regex_rule>] /
+[--oss-sharding BOOLEAN] /
+[--oss-cluster true]
+[--bigstore true]
+[--bigstore-ram-size <maximum_memory>]
+[--with-module <module_name>]
```
Comment (AlonMagrafta) [inline content/rs/references/crdb-cli-reference.md:74]: What about referencing to a page where we say which modules are supported?

File: content/rs/references/crdb-cli-reference.md [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+The `remove-instance` command deletes all data from an Active-Active instance, deletes the instance from the participating cluster, and removes the instance from the list of instances for the Active-Active database.
+
+When you remove an instance from an Active-Active database, you must specify:
+
+|Flag and argument| Argument type| Description|
+|---|---|---|
+|`crdb-guid <guid-id>`| string| The ID of the Active-Active database that you want to add the instance to|
+|`instance-id=<instance_id>`| integer| The ID of the instance to remove from the Active-Active database|
+
+{{< note >}}
```
Comment (AlonMagrafta) [inline content/rs/references/crdb-cli-reference.md:249]: Let's add to this note that this flow should be performed only when impossible to restore network.
We don't encourage this flow

File: content/rs/references/crdb-cli-reference.md [hunk 7]
```diff
@@ -0,0 +1,284 @@
...
+```sh
+# crdb-cli crdb list
+CRDB-GUID                                              NAME    REPL-ID     CLUSTER-FQDN
+e7ef4552-3705-4790-9f30-6a8e9f4bef28  shopping-cart  1        cluster1.local
+e7ef4552-3705-4790-9f30-6a8e9f4bef28  shopping-cart  2        cluster2.local
+```
+
+### Updating the configuration of an Active-Active database
+
+The `crdb update` command lets you change the configuration of an existing Active-Active database.
```
Comment (AlonMagrafta) [inline content/rs/references/crdb-cli-reference.md:153]: I'm missing here:
- crdb update is a Global configuration change which will affect all instances.
- A user can also update local configurations via rladmin, users should take this into account.

File: content/rs/references/crdb-cli-reference.md (hunk 5 above)
Comment (bmansheim) [reply content/rs/references/crdb-cli-reference.md:74]: We don't have one yet. We're working on it.

File: content/rs/references/crdb-cli-reference.md (hunk 6 above)
Comment (bmansheim) [reply content/rs/references/crdb-cli-reference.md:249]: I think that's what the note says. Can we leave it?

**Pull request merged**